.github/workflows/release.yml

```diff
@@ -27,6 +30,29 @@ jobs:
       - name: Unit Tests
         run: go test ./...
 
+      - name: update README
+        run: |
+          help_message=$(go run .)
+          echo '```' > output.txt
+          echo "$help_message" >> output.txt
+          echo '```' >> output.txt
+          sed -i '/<!-- command-line:start -->/,/<!-- command-line:end -->/{
+            /<!-- command-line:start -->/{
+              p
+              r output.txt
+            }
+            /<!-- command-line:end -->/!d
+          }' README.md
+      - name: commit and push
+        id: git-check
+        continue-on-error: true
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+          git add README.md
+          git diff-index --quiet HEAD || git commit -m "docs: update help message in README.md"
+          git push
+
```

Comment:
No, it's not.

First, the release will be triggered only manually by `workflow_dispatch`. Every *push to `master`* may update the README (with an additional commit), but will **not** trigger a release.
The Release will be triggered manually, and as so, there will be no reason to update the README on the manual trigger, because all the commits in the `master` are already updated the README if needed.

Another point, even if we will trigger the release automatically for every *push to `master`*, it will first update the README, then it will run `semantic-release`. As far as I remember, `semantic-release` will fail itself if it is not in the latest commit on a branch.
The current pipeline will stop, and the next release pipeline (triggered by the README update) will create the release.

Reply:
Removed. Will be implemented in #123